app/controllers/projects_controller.rb

```diff
@@ -120,7 +129,8 @@ def set_property_values
   end
 
   def time_period
-    (@end_date - @start_date) / 60 / 60 / 24 / 30
+    # returns time period in days
+    (@end_date - @start_date) / 60 / 60 / 24
```

Comment:
I know I gave the initial name, but we can call it `selected_time_period_days`, it's a more clear and we can remove the coment :-)